style.css

```diff
@@ -10,10 +10,14 @@ body {
   width: 20%; 
 }
 
-header {
+
+.Project {
```

Comment:
Try not to capitalize CSS class names - it's not standard CSS practice. Lowercase with hyphenation between each word is the way to go here. :)